### PR TITLE
refactor: move logic into warehouse.ai-api-client

### PR DESCRIPTION
**Pull request by JacopoDaeli**
Basic api logic moved into https://github.com/warehouseai/warehouse.ai-api-client. 
Updating this module to use the new update dependency.
No need to update tests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +27,8 @@
     "sinon": "^9.2.2"
   },
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=12.0.0",
```
Comment (rxmarbles) [inline package.json:30]: Should we at least bump this to node@14 since 12 is no longer LTS?

Comment (JacopoDaeli) [reply package.json:30]: I do not think so. Node 12 works. People still using Node 12 would get forced to update if we set this to 14.

Comment (rxmarbles) [reply package.json:30]: While true. It is no longer receiving updates. I don't think we should be expected to support folks using node 12 for this CLI.

File: src/utils/base-command.js [hunk 2]
```diff
@@ -13,10 +13,13 @@ class BaseCommand extends Command {
     this._config = new BaseCommand.Config();
     const { baseUrl, username, password } = this._config.load();
     this._request = new BaseCommand.Request({ baseUrl, username, password });
```
Comment (decompil3d) [inline src/utils/base-command.js:15]: Is `this._request` still used anywhere? Or can it be removed?

**Pull request merged**